# Time-based "M"
point_open_loop_path_generator:
  ros__parameters:
    waypoints_json: |
      [
        {
          "x": 0.0,
          "y": 0.25,
          "total_time": 8.0
        },
        {
          "x": -0.25,
          "y": 0.0,
          "total_time": 8.0
        },
        {
          "x": -0.5,
          "y": 0.25,
          "total_time": 8.0
        },
        {
          "x": -0.5,
          "y": 0.0,
          "total_time": 8.0
        }
      ]
    max_linear_speed: 0.16
    max_angular_speed: 0.9
    safety_margin: 0.1
